docs/content/foundations/appearance/border.mdx

```diff
@@ -4,110 +4,781 @@ page_title: Border
-| .border-success-darker  |     <div className="square-4 radius-sm .border-middle padding-05"><div className="square-3 radius-sm display-block border-1px border-success-darker"></div></div>     | `#206b6f` |
-| .border-info-lighter    |  <div className="square-4 radius-sm .border-middle padding-05 bg-ink"><div className="square-3 radius-sm display-block border-1px border-info-lighter"></div></div>   | `#d4e7f2` |
-| .border-info-light      |   <div className="square-4 radius-sm .border-middle padding-05 bg-ink"><div className="square-3 radius-sm display-block border-1px border-info-light"></div></div>    | `#99cae4` |
-| .border-info            |          <div className="square-4 radius-sm .border-middle padding-05"><div className="square-3 radius-sm display-block border-1px border-info"></div></div>          | `#51b8f0` |
-| .border-info-dark       |       <div className="square-4 radius-sm .border-middle padding-05"><div className="square-3 radius-sm display-block border-1px border-info-dark"></div></div>        | `#2099df` |
-| .border-info-darker     |      <div className="square-4 radius-sm .border-middle padding-05"><div className="square-3 radius-sm display-block border-1px border-info-darker"></div></div>       | `#01679d` |
-| .border-disabled-light  | <div className="square-4 radius-sm .border-middle padding-05 bg-ink"><div className="square-3 radius-sm display-block border-1px border-disabled-light"></div></div>  | `#e6e6e6` |
-| .border-disabled        |        <div className="square-4 radius-sm .border-middle padding-05"><div className="square-3 radius-sm display-block border-1px border-disabled"></div></div>        | `#c9c9c9` |
-| .border-disabled-dark   |     <div className="square-4 radius-sm .border-middle padding-05"><div className="square-3 radius-sm display-block border-1px border-disabled-dark"></div></div>      | `#adadad` |
-| .border-emergency       |       <div className="square-4 radius-sm .border-middle padding-05"><div className="square-3 radius-sm display-block border-1px border-emergency"></div></div>        | `#b60d43` |
-| .border-emergency-dark  |     <div className="square-4 radius-sm .border-middle padding-05"><div className="square-3 radius-sm display-block border-1px border-emergency-dark"></div></div>     | `#700824` |
-
-## For Designers: How to apply Border in the NCIDS Design Kit in Figma
+utility_modules:
+  - name: Border
+    utility_module_name: border
+    mixins_and_functions:
+      intro: |
+        Set width, color, style, and radius of an item’s borders.
+      outtro:
+    utility_class_display_component: 'BoxModelDemonstration'
+    utility_class_display_params:
+      groups:
+        - heading: 'Border'
+          classes:
+            - class_name: 'border'
+              additional_example_classes: 'square-8 bg-base-lightest'
```

Comment:
Please take advantage of `utilityClassDisplayParams.additional_example_classes,` is the examples below. There are too many duplicate `additional_example_classes` on each individual class. This makes the file bigger and harder to read.